app/frontend/src/components/Navigation/Navigation.tsx

```diff
@@ -212,6 +212,8 @@ export default function Navigation() {
         <SearchBox />
         <div className={classes.bug}>
           <Hidden smDown>
```

Comment:
This adds the links in the Desktop version, now to add it to the Mobile version just add the new items to `drawerItems`, as per the logout route.

Reply:
Added the links for the mobile version!

<img width="239" alt="Screen Shot 2021-04-09 at 10 59 53 AM" src="https://user-images.githubusercontent.com/43279784/114200031-cd770800-9922-11eb-827a-ca5f6307835a.png">